### ISSUE TITLE
add training iterations to all models

### ISSUE THREAD

**Issue by epicgamer17** [epicgamer17/rl-research#61]
again similar to minibatches but different since this one the model would definitely get closer to ideal results (it is training on the same data twice) so that means the loss would decrease etc (not same as doubling learning rate). 

basically its like fine tuning.
